.github/workflows/e2e-test.yml

```diff
@@ -47,28 +44,30 @@ jobs:
       - name: Install plugin
         run: docker exec -u www-data magento2-container make plugin
 
-      - name: Checkout developer module
-        uses: actions/checkout@v3
-        with:
-          repository: Adyen/adyen-magento2-dev
-          ref: 'main'
-          token: ${{ secrets.ADYEN_AUTOMATION_BOT_TEST_ACCESS_TOKEN }}
-          path: Developer
+      # The commented out section will be re-enabled with ECP-8686
+
+      # - name: Checkout developer module
+      #   uses: actions/checkout@v3
+      #   with:
+      #     repository: Adyen/adyen-magento2-dev
+      #     ref: 'main'
+      #     token: ${{ secrets.ADYEN_AUTOMATION_BOT_TEST_ACCESS_TOKEN }}
+      #     path: Developer
 
-      - name: Make the folder for dev module
-        run: docker exec -u www-data magento2-container mkdir -p /var/www/html/app/code/Adyen
+      # - name: Make the folder for dev module
+      #   run: docker exec -u www-data magento2-container mkdir -p /var/www/html/app/code/Adyen
```

Comment:
We want to keep this part in the workflow since it was actively used and will be used, we just need to skip it for now since there's an open issue in the sprint already